pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -16,8 +16,7 @@ The Python bot is tightly coupled with the Python Discord server, so to have a f
 It's possible to set the bot to use a single channel for all cogs, but that will cause extreme spam and will be difficult to work with.
 
 You can start your own server and set up channels as you see fit, but for your convenience we have a template for a development server you can use: [https://discord.new/zmHtscpYN9E3](https://discord.new/zmHtscpYN9E3).
-Keep in mind that this is not a mirror of the Python server, but a reduced version for testing purposes. A lot of the channels in the Python server were merged.
-
+Keep in mind that this is not an exact mirror of the Python server, but a reduced version for testing purposes.
```

Comment:
I'm not too sure about that either, because If I understood right, we'll be revamping the test server as well so that it reflects the necessary channels for the bot to work.
AFAIK, we've only condensed the test server channels to avoid the config hell that we used to go through, which will soon change.

What I meant by "this is not an exact mirror" is the fact that not ALL channels from prod are here since they're not needed for the bot to work, but maybe we should work on that rewording, WDYT ?

Reply:
I meant as an addition to what was already there, should've been clearer.